File: beginner/pagina12/3147/3147.js
const fs = require('fs')
const input = fs.readFileSync('/dev/stdin', 'utf8');
const [h, e, a, o , w, x] = input.split(' ').map(Number)

if( (h + a + e + x) >= (o + w) ){
    console.log("Middle-earth is safe.")
} else {
    console.log("Sauron has returned.")
}

File: beginner/__3000__/3147/3147.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
		
        Scanner input = new Scanner(System.in);

        String[] data = input.nexLine().split(" ");

        int h = Integer.parseInt(data[0]);
        int e = Integer.parseInt(data[1]);
        int a = Integer.parseInt(data[2]);
        int o = Integer.parseInt(data[3]);
        int w = Integer.parseInt(data[4]);
        int x = Integer.parseInt(data[5]);

        if( (h + a + e + x) >= (o + w) ){
            System.out.println("Middle-earth is safe.");
        } else {
            System.out.println("Sauron has returned.");
        }


        input.close();
	}
}